setuptools/config.py

```diff
@@ -301,7 +301,21 @@ def _parse_attr(cls, value):
         module_name = '.'.join(attrs_path)
         module_name = module_name or '__init__'
 
-        sys.path.insert(0, os.getcwd())
+        parent_path = os.getcwd()
+        if package_dir:
+            if attrs_path[0] in package_dir:
+                # A custom path was specified for the module we want to import
+                custom_path = package_dir[attrs_path[0]]
+                parts = custom_path.rsplit('/', 1)
```

Comment:
There probably should be OS directory separator instead of hardcoded `/`.

Reply:
The [setuptools documentation](https://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files) specifies that Unix-style separators should be used for all paths in setup.cfg, in order to facilitate portability (otherwise package builds would fail on platforms using the other separator type).

> Also notice that if you use paths, you must use a forward slash (/) as the path separator, even if you are on Windows. Setuptools automatically converts slashes to appropriate platform-specific separators at build time.

Reply:
Yes, it is so, and the last sentence is what I was concerned with: I've suspected this automatic normalization is done after the parsing stage, but was not sure.

Reply:
I'm pretty sure it's after; everything up to this point is mostly config file parsing, without much knowledge of what the individual values represent.  Also, AppVeyor builds passed on Windows with the new test cases including forward slashes in `package_dir`.